signer/core/cliui.go

```diff
@@ -243,25 +243,30 @@ func (ui *CommandlineUI) OnApprovedTx(tx ethapi.SignTransactionResult) {
 	}
 }
 
-func (ui *CommandlineUI) OnSignerStartup(info StartupInfo) {
+func (ui *CommandlineUI) ShowAccounts() string {
```

Comment:
Myeah, sure, that's one way of doing it. If you do this, then you should make it lowercase - so it's a non-public method `showAccounts`. 


But the primary thing I wanted was to make it async. So instead of just calling
```
	addresses := ui.ShowAccounts()
	fmt.Print("Accounts known to Clef:\n\n")
	fmt.Printf("%s", addresses)
```
You would do 
```
	go fun(){
		addresses := ui.ShowAccounts()
		fmt.Print("Accounts known to Clef:\n\n")
		fmt.Printf("%s", addresses)
	}()
```

Reply:
ah, yes ok: https://github.com/ethereum/go-ethereum/pull/26082/commits/c1dbc024828cab56d6a06fd91f20db435842157d